lib/collection.js

```diff
@@ -366,7 +396,7 @@ Collection.prototype._parseDefinition = function _parseDefinition(definition) {
   var self = this;
 
   // Hold the Schema
-  this.schema = _.cloneDeep(definition.definition);
+  this.schema = structuredClone(definition.definition);
```

Comment:
FYI this works only in new node versions, can we change it back to normal clone, example databench is running with old version

Reply:
all should be upgraded by now - if not, will plan an upgrade